test/e2e/managed_test.go

```diff
@@ -61,3 +61,14 @@ func TestEditManagedRoute(t *testing.T) {
 		t.Fatalf("error: %s", err)
 	}
 }
+
```

Comment:
This test should be in a new file.
The existing ones could prob be better named 

```bash
managementstate_managed_test.go
managementstate_unmanaged_test.go
managementstate_removed_test.go
```
I may suggest creating:

```bash
branding_operator_config_test.go
branding_managed_config_test.go
```
or even just:

```bash 
branding_test.go
```
Either way is fine, just so long as its clear these features do not overlap.

Reply:
If I were to suggest an approach for the simpler test, I'd go with something like:

```golang
// branding_test.go
// the simpler test first would be branding set via the operator config
func TestOperatorConfigBranding(t *testing.T) {
  online := "online"
  client := testframework.MustNewClientset(t, nil)
  // defer the cleanup
  // we should prob have a Reset() func, to just put the operator config back to defaults
  defer testframework.Reset(t, client, operatorsv1.Normal)
  // helper to update the operator config 
  err := testframework.SetBrand(t, client, online)
   if err != nil {
      t.Fatalf("error: %s", err)
   }
   // now check the console-config in openshift-console & verify it has brand set
  cm, err := testframework.GetConsoleConfigmap(client)
   if err != nil {
      t.Fatalf("error: %s", err)
   }
   if !brandIs(online) {
      t.Fatalf("error: brand was not set to %v", online)
   }
}

// after the above, can do the more complex branding change via the 
// configmap in openshift-config-managed
TestBrandingViaManagedConfigmap() {
   // TODO: later, after the first test works
}
```
`pkg/testframework/console.go` (note that #206 will rename this file to `console-operator.go`)

```golang
// helper funcs
func SetBrand(t *testing.T, client *Clientset, brand string) error {
   // need the config	
   operatorConfig, err := client.Consoles().Get(consoleapi.ConfigResourceName, metav1.GetOptions{})
   if err != nil {
      return err
   }
   // set it
   operatorConfig.Spec.Customization.Brand = brand
   err := client.Consoles().Update(operatorConfig)
   if err != nil {
      return err
   }
   return nil   
}
```
Untested as I wrote this in github, but its a direction.

Reply:
+1 for `branding_test.go`